src/libawkward/Content.cpp

```diff
@@ -243,4 +243,15 @@ namespace awkward {
     }
   }
 
+  const int64_t Content::axis_wrap(int64_t axis) const {
+    int64_t mindepth = minmax_depth().first;
+    int64_t maxdepth = minmax_depth().second;
+    int64_t depth = purelist_depth();
+    if (axis < 0  &&  mindepth == depth  &&  maxdepth == depth) {
+      return (depth - 1 + axis);
+    }
+    else {
+      return axis;
+    }
+  }
```

Comment:
Maybe `negative_axis` to indicate what it handles?

Reply:
Oh, "wrap", as in "wrap around from negative to non-negative." I didn't get the meaning at first—I thought it was a generic "wrapper," a word that I'd use internally in the midst of an implementation because I can't think of a word.

Then maybe `wrap_negative_axis`?

Reply:
The ```axis``` can be positive

Reply:
`axis_wrap_if_negative`?

Reply:
`axis_wrap_if_negative`?

Reply:
done